appveyor.yml

```diff
@@ -143,20 +131,30 @@ install:
   # Check NumPy, etc. installed correctly
   - cmd: pip list
 
-  - cd %NUPIC_CORE%
-  - mkdir %NUPIC_CORE%\build\
-  - mkdir %NUPIC_CORE%\build\release
-  - mkdir %NUPIC_CORE%\build\scripts
+  # Make unix-compatible patch.exe available to the build by copying it from
```

Comment:
Reviewer: I reorganized this build script to make it easier for me to debug an issue. I am inclined to leave the change in if there are no objections.